infrastructure/lambda/template.yaml

```diff
@@ -797,61 +744,7 @@ Resources:
 
 Outputs:
   AddressApiBaseUrl:
+    Description: "Base url of the Address CRI API"
     Value: !Sub "https://${AddressApi}.execute-api.${AWS::Region}.amazonaws.com/${Environment}/"
     Export:
       Name: !Sub ${AWS::StackName}-AddressApiBaseUrl
-
```

Comment:
I've dropped all these outputs apart from the base url as they're not necessary - please argue the case to keep some or all though! 😄